model-optimizer/extensions/back/TopKNormalizer.py

```diff
@@ -58,6 +58,16 @@ def replace_pattern(graph: Graph, match: dict):
                                                    {'override_output_shape': True})
         node.in_port(1).get_connection().insert_node(reshape)
 
+        TopKNormalizer.normalize_outputs(node, graph)
+
+    @staticmethod
+    def normalize_outputs(node: Node, graph: Graph = None):
```

Comment:
Remove the `graph` parameter. Get the `graph` from the `node`.

Reply:
Done.